test/node/object_id_tests.js

```diff
@@ -1,5 +1,4 @@
 'use strict';
-
 const Buffer = require('buffer').Buffer;
```

Comment:
I think we should keep the new line between `'use strict'` and the `require` statements that follow, and delete the line between the `@ignore` and the following `it` block. Even better, the `@ignore` blocks can be removed entirely, since they serve no purpose here (Neal tells me it was for old jsdoc which was never even ran on the test files).